Health/Presentation/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,150 @@
+        ),
+         ProfileCellModel(
+            title: "일반 설정",
+            iconName: "gearshape.fill",
+            isSwitch: false
+         )
+        ],
+        [ProfileCellModel(
+            title: "Apple 건강 앱",
+            iconName: "applelogo",
+            isSwitch: true,
+            switchState: true
+        )
+        ]
+    ]
+    
+    override func setupAttribute() {
+        super.setupAttribute()
+        
+        applyBackgroundGradient(.midnightBlack)
+        
+        navigationItem.title = "프로필"
+        
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "profileCell")
+        tableView.rowHeight = 68
+        tableView.backgroundColor = .clear
+        
+        
+        
+    }
+    
+    
+}
+
+extension ProfileViewController: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return sectionItems.count
+    }
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return sectionTitles[section]
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return sectionItems[section].count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "profileCell", for: indexPath)
+        let model = sectionItems[indexPath.section][indexPath.row]
+        
+        var content = cell.defaultContentConfiguration()
+        content.text  = model.title
+        content.image = UIImage(systemName: model.iconName)
+        content.imageProperties.tintColor = .systemGray
+        cell.contentConfiguration = content
+        cell.backgroundColor = UIColor.buttonText.withAlphaComponent(0.1)
+        
+        
+        if model.isSwitch {
+            let toggle = UISwitch(frame: .zero)
+            toggle.isOn = model.switchState
+            toggle.tag = indexPath.section
+            toggle.onTintColor = .accent
+            toggle.addTarget(self, action: #selector(switchChanged(_:)), for: .valueChanged)
+            cell.accessoryView = toggle
+            cell.selectionStyle = .none
+        }
+        
+        return cell
+    }
+    
+}
+extension ProfileViewController: UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView,
+                   didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+        
+        let model = sectionItems[indexPath.section][indexPath.row]
+        
+        switch (indexPath.section, indexPath.row) {
+        case (0, 0):
+            performSegue(withIdentifier: "bodyInfo", sender: nil)
+            break
+        case (1, 0):
+            break
+        case (1, 1):
+            break
+        default:
+            break
+        }
```

Comment:
위치 기반보다는 타이틀 같은걸로 구분하면 좋을 것 같아요

```suggestion
switch model.title {
    case "신체 정보":
        performSegue(withIdentifier: "bodyInfo", sender: nil)
    case "목표 걸음 설정":
        ...
    default:
        break
}
```